src/ds++/FMA.hh

```diff
@@ -7,16 +7,34 @@
  * \note    Copyright (C) 2017-2018 Los Alamos National Security, LLC.
  *          All rights reserved.
  *
- * Intel Haswell and later (and also modern AMD cpus), have hardware FMA
- * features. Using \c fma(c,b,a) is more accurate than \c a*b+c because there is
- * less roundoff error. This difference in accuracy causes issues for IMC
- * because solutions are not consistent between older and newer hardware.
+ * Intel Haswell and later (and also modern AMD cpus) have hardware FMA
+ * features. On machines without hardware FMA, the use of \c fma(c,b,a) provides
+ * a accurate value than \c a*b+c because roundoff error is accounted for.
+ * However, \c fma can be slow (maybe 3x slower).
  *
  * To improve solution consistency across platforms, we can choose to use a
  * software-based infinite precision fma call.
  *
  * \note Intel's code for detecting FMA availability on hardware.
  * https://software.intel.com/en-us/node/405250?language=es&wapkw=avx2+cpuid
+ *
+ * \note The Intel compiler doesn't seem to set \c FP_FAST_FMA. If you know that
+ * your algorithm is sensitive to roundoff, you might to locally disable
```

Comment:
might to locally -> might need to locally

Reply:
Thanks for catching this.